### ISSUE TITLE
About Reg Loss in AttentionGAN-v1

### ISSUE THREAD

**Issue by DISAPPEARED13** [Ha0Tang/AttentionGAN#26]
I just use AttentionGAN-V1 combined with my GAN model, and my reg loss decrease to about 0,165 and then stay still, 

looking the `attention_mask` variable I found that after iterations and epochs, all voxels stay about 0.26  with not any attention i need (can't detect any domain gap?? after training) and then stop update each voxel's probability (from 0 to 1), 

but the result did better for non-attention situation, 
could you guide me how to fix this or its really normal to this situation?

In my opinion I think, though GAN maybe stable after epochs but attention from generator may still catch the difference between domains, and highlight the main different part.

@Ha0Tang , please and Thanks alot, best wishes.
